Kosystem.Shared/RoomModel.cs

```diff
@@ -1,17 +1,47 @@
-﻿namespace Kosystem.Shared
+using System;
+
+namespace Kosystem.Shared
 {
-    public class RoomModel
+    public readonly struct RoomModel : IEquatable<RoomModel>
     {
-        public int Id { get; set; }
-
-        public string Name { get; set; }
+        public int Id { get; }
 
-        public PersonModelCollection People { get; } = new PersonModelCollection();
+        public string Name { get; }
 
         public RoomModel(int id, string name)
         {
             Id = id;
             Name = name;
         }
+
+        public RoomModel Update(UpdateRoomModel update)
```

Comment:
Codacy found an issue: [Rename the parameter 'update' so that it does not duplicate the method name.](https://app.codacy.com/gh/HelloWorldSweden/Kosystem/pullRequest?prid=6010716)